internal/internal_worker.go

```diff
@@ -122,6 +120,9 @@ type (
 
 	// workerExecutionParameters defines worker configure/execution options.
 	workerExecutionParameters struct {
+		// Domain name.
```

Comment:
Add info about the default value

Reply:
This is internal struct and it doesn't have default value. Public structs have default value in comment.